src/Ownership/Commands/ApproveOwnership.php

```diff
@@ -10,21 +10,21 @@
 final class ApproveOwnership
 {
     private UUID $id;
-    private UserId $approverId;
+    private UserId $requesterId;
 
     public function __construct(UUID $id, UserId $requesterId)
```

Comment:
I added it because an ownership request can be done for someone else. As an admin, I can request the ownership of an item for someone else. But for approving or rejecting it is only the action itself that needs to be guarded with the current user and that is validated inside the HTTP request handler.